workspace-run.js

```diff
@@ -108,6 +106,7 @@ run({ name: 'yarn', args: ['workspaces', '--silent', 'info'] })
 			},
 			{ commands: [], packages: [] },
 		);
+		console.log(orderedWorkspaceInfo);
```

Comment:
do you want to keep it ?. I would keep only the packages 
![Screenshot 2021-09-22 at 10 04 15](https://user-images.githubusercontent.com/19857479/134305965-a7c13971-f779-44a6-9ed1-aaa1614fe510.png)